scipy/spatial/tests/test_spherical_voronoi.py

```diff
@@ -76,14 +76,14 @@ def test_vertices_regions_translation_invariance(self):
         sv_origin = SphericalVoronoi(self.points)
         center = np.array([1, 1, 1])
         sv_translated = SphericalVoronoi(self.points + center, center=center)
-        assert_array_equal(sv_origin.regions, sv_translated.regions)
+        assert_equal(sv_origin.regions, sv_translated.regions)
```

Comment:
here and below, `assert_equal` is used for lists-of-lists rather than converting to ndarrays